corehq/apps/translations/app_translations.py

```diff
@@ -826,34 +826,38 @@ def _update_case_list_translations(sheet, rows, app):
     detail_tab_headers = [None for i in module.case_details.long.tabs]
     index_of_last_enum_in_condensed = -1
     index_of_last_graph_in_condensed = -1
+
+    def remove_description_from_case_property(row):
+        return re.match('.*(?= \()', row['case_property']).group()
```

Comment:
Nice regex. I like how you took advantage of the fact that `*` is greedy to get everything before the last open paren.

I second the suggestion of a test. Not sure how @snopoke would feel about a doctest. I believe doctests are docs not tests, but in this case it might help to show, close to the code and in just a few lines, how that regex works and when it errors (if `row` doesn't contain `" ("`).

Reply:
Thanks!
I found a spot that looked reasonable in existing unit tests instead of going the doctests approach: https://github.com/dimagi/commcare-hq/pull/23564/commits/9ca0fbdf0e3fe5f8956b04991b324dca4bace4bc.

This function only gets called when the description is in parens at the end of the case property, ie: 
```
        elif row['case_property'].endswith(" (graph series config)"):
            row['id'] = _remove_description_from_case_property(row)
```
I tried to make the name of the function reflect this. Think that's a good approach?